src/AppMenuSearch.cc

```
@@ -380,17 +393,31 @@ static int calculateFuzzyScore(const QString &pattern, const QString &text)
     int consecutive = 0;
     int prevMatchIdx = -1;
 
-    for (int textIdx = 0; textIdx < textLen && patternIdx < patternLen; ++textIdx) {
-        const QChar pChar = pattern.at(patternIdx).toLower();
-        const QChar tChar = text.at(textIdx).toLower();
+    const QString pLower = patternLower.isEmpty() ? pattern.toLower() : patternLower;
```

Clang (14.0.6)

[warning] variable 'pLower' is not initialized

(cppcoreguidelines-init-variables)